src/ArrayUtils/MergeReplaceKey.php

```diff
@@ -6,15 +6,11 @@
 
 final class MergeReplaceKey implements MergeReplaceKeyInterface
 {
-    /** @var mixed */
-    protected $data;
-
     /**
      * @param mixed $data
      */
-    public function __construct($data)
+    public function __construct(protected $data)
```

Comment:
`mixed` can be explicit here

Reply:
Ok, I applied `mixed` param type on https://github.com/laminas/laminas-stdlib/pull/75/commits/d1085083254efd023f628338ccb1fe2a8ca00a26